nifi-nar-bundles/nifi-azure-bundle/nifi-azure-processors/src/main/java/org/apache/nifi/processors/azure/eventhub/ConsumeAzureEventHub.java

```diff
@@ -332,6 +345,24 @@ public void setWriterFactory(RecordSetWriterFactory writerFactory) {
                     .valid(false)
                     .build());
         }
+
+        if (storageAccountKey == null && storageSasToken == null) {
+            results.add(new ValidationResult.Builder()
+                    .subject("Storage Account Key or Storage SAS Token")
+                    .explanation(String.format("Either %s or %s should be set.",
```

Comment:
The explanation will be concatenated into the middle of the validation failure message so it should start with lower case (_"...is invalid because either..."_)